src/languages/en.ts

```diff
@@ -2773,7 +2773,8 @@ const translations = {
             descriptionHint: 'Share information about this workspace with all members.',
             welcomeNote: 'Please use Expensify to submit your receipts for reimbursement, thanks!',
             subscription: 'Subscription',
-            markAsExported: 'Mark as manually entered',
+            markAsEntered: 'Mark as manually entered',
+            markAsExported: 'Mark as manually exported',
```

Comment:
Not your code, but we should rename `unhold: 'Unhold'` to `unhold: 'Remove hold'`